src/renderer/components/app.tsx

```diff
@@ -49,7 +48,6 @@ import { nodesStore } from "./+nodes/nodes.store";
 import { podsStore } from "./+workloads-pods/pods.store";
 import { sum } from "lodash";
 
-@observer
```

Comment:
Removing this will break reactivity from `renderExtensionTabLayoutRoutes()` and `renderExtensionRoutes()`

Reply:
The rule of `@observer`: if some observables used in direct `render()` of component then it needs to used.

Reply:
How to verify this?

Reply:
Nevermind, I do see that routes won't get properly registered if an extension is enabled after app has started.

Reply:
Look at the component ~~as I did~~ to see if it's use _any_ observables :)

Maybe this is root of issues:
```
          const pageComponent = () => <page.components.Page/>;

          return <Route key={`extension-tab-layout-route-${index}`} path={page.url} component={pageComponent}/>;
```

With `const pageComponent = () => <page.components.Page/>;` in `render()` the component (arrow-function) re-created every time.

Should be like `<Route path={page.url} component={page.components.Page}/>`

Reply:
Indeed that fixes the issue 🏆 Will push update asap.

Reply:
Fixed.